src/gmail.js

```diff
@@ -578,24 +578,23 @@ var Gmail = function(localJQuery) {
 
 
     api.get.unread_inbox_emails = function() {
-        var dom = $("div[role=navigation]").find("[title*='" + api.tools.i18n("inbox") + "']");
-
-        if(dom.length > 0) {
-            if(dom[0].title.indexOf("(") !== -1) {
-                return parseInt(dom[0].title.split(":")[0].replace(/[^0-9]/g, ""));
+            var dom = $("div[role=navigation]").find("[title*='" + api.tools.i18n("inbox") + "']");
+            if(dom.length > 0) {
+                if(dom[0].title.indexOf("Inbox") !== -1) {
+                    return parseInt(dom[0].attributes['aria-label'].value.replace(/[^0-9]/g, ""));
+                }
             }
```

Comment:
Different space/tab settings makes this diff a bit messy and hard to read. Can you fix that?